### PR TITLE
enhancement:  address-Book implemented

### PR DESCRIPTION
**Pull request by Daljit-fleetHawks**
## Change Description

Customer/Broker/Vendor/Shipper/Consignee/Staff/FC-Companies/Carrier forms and list updated except address column.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/pages/fleet/drivers/driver-list/driver-list.component.ts [hunk 1]
```diff
@@ -101,7 +111,7 @@ export class DriverListComponent implements OnInit, OnDestroy {
               this.drivers.push(iterator);
             }
           }
-
+          console.log('driver', drivers);
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/drivers/driver-list/driver-list.component.ts]: remove console.logs

File: src/app/pages/fleet/geofence/add-geofence/add-geofence.component.ts [hunk 2]
```diff
@@ -243,7 +243,7 @@ export class AddGeofenceComponent implements OnInit {
       .getData('geofences/' + this.getGeofenceID)
       .subscribe((result: any) => {
         result = result.Items[0];
-        // console.log('result', result);
+        console.log('result', result);
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/geofence/add-geofence/add-geofence.component.ts]: please remove console.logs

File: src/app/pages/fleet/maintenance/service-log/add-service/add-service.component.ts [hunk 3]
```diff
@@ -295,11 +295,14 @@ export class AddServiceComponent implements OnInit {
...
-      this.reminders.forEach(element => {
-          element.buttonShow = false;
+      let response = await result.Items;
+      response.forEach(element => {
+          if(element.reminderType === 'service') {
+            this.reminders.push(element);
+            element.buttonShow = false;
+          }
       });
+      console.log("reminder", this.reminders);
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/maintenance/service-log/add-service/add-service.component.ts]: please review console.logs

File: src/app/pages/fleet/maintenance/service-log/service-detail/service-detail.component.ts [hunk 4]
```diff
@@ -104,6 +104,7 @@ export class ServiceDetailComponent implements OnInit {
     this.apiService.getData('vehicles/get/list')
       .subscribe((result: any) => {
         this.vehiclesObject = result;
+        console.log('vehiclesObject', this.vehiclesObject);
       });
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/maintenance/service-log/service-detail/service-detail.component.ts:108]: please remove console.logs

**Review by kunalfleethawks — CHANGES_REQUESTED** [review]
Please do the requested changes

File: src/app/pages/fleet/maintenance/service-log/service-detail/service-detail.component.ts [hunk 5]
```diff
@@ -104,6 +104,7 @@ export class ServiceDetailComponent implements OnInit {
     this.apiService.getData('vehicles/get/list')
       .subscribe((result: any) => {
         this.vehiclesObject = result;
+        console.log('vehiclesObject', this.vehiclesObject);
```
Comment (amritfleethawks) [inline src/app/pages/fleet/maintenance/service-log/service-detail/service-detail.component.ts]: please remove this..

**Review by amritfleethawks — APPROVED** [review]

**Review by Daljit-fleetHawks — COMMENTED** [review]
changes done

**Review by kunalfleethawks — APPROVED** [review]
Looks good.

**Pull request merged**